### ISSUE TITLE
Rename second argument in Option.fold

### ISSUE THREAD

**Issue by georgeci** [MarioAriasC/funKTionale#24]
The name of the second argument "f" like a not very good solution. I like to use named argument for fold calling and any more functions.
Of course, I can write:
```
 fold(isEmpty = {...}){value -> ...}
```
But I want to be able to write like this
```
 fold(isEmpty = {...},  some = {value -> ...})
```

**Comment by MarioAriasC** [comment]
I used `f` as convention on everything that is function; I could change it in a future release

**Comment by MarioAriasC** [comment]
fix on 1.2
